Cargo.lock

```diff
@@ -2,7 +2,10 @@
 name = "rstreediff"
 version = "0.1.0"
 dependencies = [
+ "dot 0.1.2 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
I must admit, I tend to think that this only makes sense at the point that one makes releases. At the moment if I e.g. decide to use Rust nightly and you use Rust stable -- and the code compiles under both -- I think the `Cargo.lock` can end up looking different for both of us. I might be wrong.